app/mailers/app_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class AppMailer < ActionMailer::Base
+  def send_welcome_email(user)
+    @user = user
+    mail to: user.email, from: "info@myflix.com", subject: "Welcome to Myflix"
```

Comment:
The `from:` option could be refactored to `default from: ...` callback. Check documentation for this.